boards/arm/stm32h747i_disco/stm32h747i_disco_m7.dts

```diff
@@ -165,6 +170,26 @@
 	};
 };
 
+zephyr_udc0: &usbotg_hs {
+	pinctrl-0 = <&usb_otg_hs_ulpi_ck_pa5
+				&usb_otg_hs_ulpi_d0_pa3
+				&usb_otg_hs_ulpi_d1_pb0
+				&usb_otg_hs_ulpi_d2_pb1
+				&usb_otg_hs_ulpi_d3_pb10
+				&usb_otg_hs_ulpi_d4_pb11
+				&usb_otg_hs_ulpi_d5_pb12
+				&usb_otg_hs_ulpi_d6_pb13
+				&usb_otg_hs_ulpi_d7_pb5
+				&usb_otg_hs_ulpi_stp_pc0
+				&usb_otg_hs_ulpi_dir_pi11
+				&usb_otg_hs_ulpi_nxt_ph4>;
```

Comment:
If these pins should be set in high-speed, it would make sense to do it directly in hal_stm32 module (modify ulpi pins generation to be high speed by default)

Reply:
I already have a PR for this within the hal_stm32. I just need to update the module hash to point to the updated commit once that is merged in
https://github.com/zephyrproject-rtos/hal_stm32/pull/155